src/test/java/com/devonfw/sample/archunit/LayerDependencyRulesTest.java

```diff
@@ -0,0 +1,97 @@
+package com.devonfw.sample.archunit;
+
+import com.tngtech.archunit.core.importer.ImportOption;
+import com.tngtech.archunit.junit.AnalyzeClasses;
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.lang.ArchRule;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.noClasses;
+
+@AnalyzeClasses(packages = "com.devonfw.sample.archunit", importOptions = ImportOption.DoNotIncludeTests.class)
+public class LayerDependencyRulesTest {
```

Comment:
If you put the rules which layer can access which other layer into this separate class the `ArchitectureTest` remains pointless as it more or less does nothing and can never fail or do I miss something?

Reply:
No, you did not miss anything. 
I left the ArchitectureTest in case you have planned something additional with it, except serving as a template.

With respect to your concerns, I updated ArchitectureTest and deleted LayerDependencyRulesTest since it is not needed now anymore.

Reply:
Thanks, done and resolved.